data_and_models/pipelines/ner/dvc.yaml

```diff
@@ -88,14 +88,14 @@ stages:
     foreach: ${entities}
     do:
       cmd: >-
-        python eval.py
+        python eval_ner.py
```

Comment:
Is this because of mypy complaining about modules with the same names?

Reply:
Indeed, there are several `eval.py` into `data_and_models` folder. `Mypy` is complaining about it.